sdk/iOS/src/MSSyncContextInternal.h

```diff
@@ -27,6 +28,8 @@
 
 -(NSOperation *) pullWithQuery:(MSQuery *)query queryId:(NSString *)queryId completion:(MSSyncBlock)completion;
 
+-(NSOperation *) pullWithQuery:(MSQuery *)query queryId:(NSString *)queryId pullSettings:(MSPullSettings *)pullSettings completion:(MSSyncBlock)completion;
```

Comment:
Settings is too generic. I was planning of calling it sync settings, but not all the pull settings would be shared with push. Later we can have PullSettings and PushSettings derive from SyncSettings if needed.